content/sensu-go/5.5/reference/tessen.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Tessen"
+description: "Tessens sends anonymized data about the Sensu instance to Sensu Inc. You can use sensuctl to view and manage the Tessen configuration. Read the reference to configure Tessen."
+weight: 10
+version: "5.5"
+product: "Sensu Go"
+menu: 
+  sensu-go-5.5:
+    parent: reference
+---
+
+- [Configuring Tessen](#configuring-tessen)
+- [Specification](#tessen-specification)
+- [Example](#tessen-example)
+
+## What is Tessen?
+
+Tessen is the Sensu call-home service.
+Included with every Sensu backend, Tessen sends anonymized data about the Sensu instance to Sensu Inc. every six hours, including:
+
+- The Sensu version
+- The number of Sensu agents and backends in the cluster
+- Whether the Sensu instance uses an [enterprise license][4]
+
+All data submissions are logged for complete transparency and transmitted over HTTPS.
+[Read the blog post][1] to learn more about Tessen.
+
+## Configuring Tessen
+
+Tessen is enabled by default.
+You can use the [Tessen API][2] and [sensuctl][3] to view and manage the Tessen configuration.
+[Licensed][4] Sensu instances cannot opt out of Tessen.
+
+To see the status of Tessen:
+
+{{< highlight shell >}}
+tessen info
+{{< /highlight >}}
+
+To opt-out of Tessen:
+
+{{< highlight shell >}}
+tessen opt-out
+{{< /highlight >}}
+
+To opt-in to Tessen:
+
+{{< highlight shell >}}
+tessen opt-in
+{{< /highlight >}}
+
+## Tessen specification
+
+### Top-level attributes
+
+type         | 
+-------------|------
+description  | Top-level attribute specifying the [`sensuctl create`][sc] resource type. Tessen configuration should always be of type `TessenConfig`.
+required     | Required for Tessen configuration in `wrapped-json` or `yaml` format for use with [`sensuctl create`][sc].
+type         | String
+example      | {{< highlight shell >}}"type": "TessenConfig"{{< /highlight >}}
+
+api_version  | 
+-------------|------
+description  | Top-level attribute specifying the Sensu API group and version. For Tessen configuration in Sensu backend version 5.5, this attribute should always be `core/v2`.
+required     | Required for Tessen configuration in `wrapped-json` or `yaml` format for use with [`sensuctl create`][sc].
+type         | String
+example      | {{< highlight shell >}}"api_version": "core/v2"{{< /highlight >}}
+
+metadata     | 
+-------------|------
+description  | Empty, top-level scope
+required     | Required for Tessen configuration in `wrapped-json` or `yaml` format for use with [`sensuctl create`][sc].
```

Comment:
Is the metadata scope required?

Reply:
Nope, the metadata scope should be empty. 

![giphy](https://user-images.githubusercontent.com/10634078/55426639-26dd1d00-553a-11e9-9d48-14e5670055f5.gif)

Reply:
So would we expect this to work with `sensuctl create -f` (no metadata scope)?

```
{
  "type": "TessenConfig",
  "api_version": "core/v2",
  "spec": {
    "opt_out": false
  }
}
```

Reply:
Yup!